desk/mar/chat/heads-2.hoon

```diff
@@ -0,0 +1,14 @@
+/-  c=chat
+/+  j=chat-json
+|_  =chat-heads:c
+++  grad  %noun
+++  grow
+  |%
+  ++  noun  chat-heads
+  ++  json  (chat-heads:enjs:j chat-heads)
```

Comment:
Same here - it would be better to version json encoders too, especially since they might be silently updated. (How long since anyone had looked at mark-warmer?) As they are all in the same file for now,  we would just need to put in a version anchor at top level.
```suggestion
  ++  json  (chat-heads:v9:enjs:j chat-heads)
```